ethicalads-theme/templates/ea/publishers.html

```diff
@@ -81,10 +81,10 @@ <h3>No Cookies</h3>
         </div>
 
         <!-- Heading -->
-        <h3>Publisher Friendly</h3>
+        <h3>Talk to a human</h3>
 
         <!-- Text -->
-        <p class="text-gray-800 mb-0">We started as a publisher, and still run ethical ads on all our sites. We believe in supporting open source, because we're open source devs ourselves.</p>
+        <p class="text-gray-800 mb-0">Our ad network is run by people who understand web publishing and open source. <strong>A real human is always 1 email away.</strong></p>
```

Comment:
```suggestion
        <p class="text-gray-800 mb-0">Our ad network is run by people who understand web publishing and open source. Unlike other networks, <strong>a real human is 1 email away.</strong></p>
```

Don't feel strongly here but just a thought.